samples/correlation-csharp/FunctionAppCorrelation/FanOutFanIn.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionAppCorrelation
+{
+    public class FanOutFanIn
+    {
+        [FunctionName(nameof(FanOutFanInOrchestrator))]
+        public static async Task<string[]> FanOutFanInOrchestrator(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var tasks = new Task<string>[3];
+            tasks[0] = context.CallActivityAsync<string>(nameof(FanOutFanIn_Hello), "Tokyo");
+            tasks[1] = context.CallActivityAsync<string>(nameof(FanOutFanIn_Hello), "Seattle");
+            tasks[2] = context.CallActivityAsync<string>(nameof(FanOutFanIn_Hello), "London");
+            return await Task.WhenAll(tasks);
+        }
+
+        [FunctionName(nameof(FanOutFanIn_Hello))]
+        public static string FanOutFanIn_Hello([ActivityTrigger] string name, ILogger log)
+        {
+            log.LogInformation($"Saying hello to {name} at the same time.");
+            return $"Hello {name}!";
+        }
+
+        [FunctionName(nameof(HttpStart_FanOutFanIn))]
+        public async Task<IActionResult> HttpStart_FanOutFanIn(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get")]
+            HttpRequest req,
+            [DurableClient] IDurableOrchestrationClient starter,
+            ILogger log)
+        {
+            string instanceId = await starter.StartNewAsync(nameof(FanOutFanInOrchestrator), null);
+
+            log.LogInformation($"Started orchestration with ID = '{instanceId}'.");
+            return starter.CreateCheckStatusResponse(req, instanceId);
+        }
+    }
+}
```

Comment:
I see lot of samples like this one, where the orchestration does not appear to have anything `telemetry` specific. Would it be possible to re-use the already existing samples and to provide a flag to enable correlation tracing on them? I'm just trying to avoid duplicated code

Reply:
We could use compilation tricks to have a common orchestration defined in one place, but include it in multiple projects. The main difference between the files appears to be the namespace, and I don't know if that matters.

Reply:
Moving to ToDos issue